google/cloud/forseti/common/gcp_api/securitycenter.py

```diff
@@ -93,9 +92,7 @@ def __init__(self, **kwargs):
         # pylint: disable=protected-access
         if kwargs.get('gcp_service')._resourceDesc.get('version') == 'v1beta1':
             component = 'organizations.sources.findings'
-        else:
-            # alpha api
-            component = 'organizations.findings'
+
```

Comment:
Do you think that we still need the above `if` statement to check if this is a beta api?  I am inclined to think that we can remove it, and rely on this as the default behavior.